### PR TITLE
Add 99 confidence

### PR DESCRIPTION
**Pull request by oskarth**
cc @rasom

### CODE REVIEW

File: ideas/99-confidence.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    Status: Draft
+    Created: 2018-03-22
+
+
+## Summary
+As end users we currently don't have confidence that messages are being sent, delivered and read. This swarm aims to address this.
+
+## Swarm Participants
+- Lead Contributor: @rasom
+- Testing & Evaluation: <!-- @username -->
```
Comment (oskarth) [inline ideas/99-confidence.md]: cc @annadanchenko for evaluator - maybe UXR can assist here? @chadyj WDYT?

File: ideas/99-confidence.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+- Contributor: <!-- @username -->
```
Comment (oskarth) [inline ideas/99-confidence.md]: IIRC:

1 Clojure
1 Go
1 Design
1 UXR
1 PM

Missing / too much?

File: ideas/99-confidence.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    b. Automated switching?
+    c. How ensure no switching is necessary? (HA, see 4)
+
+### Requirements & Dependencies
+
+Request messages history after background: https://github.com/status-im/status-react/pull/3493
+New protocol https://github.com/status-im/status-react/pull/3273
+https://github.com/status-im/ideas/issues/87
+
+### Minimum Viable Product
```
Comment (oskarth) [inline ideas/99-confidence.md]: I've seen iteration scope in a few places, google docs, slack PM and slack channel - can we make it more precise what we think is reasonable for next ~1w (before next Town Hall)?

File: ideas/99-confidence.md (hunk 2 above)
Comment (oskarth) [reply ideas/99-confidence.md]: What is minimum to get started?

File: ideas/99-confidence.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+Probably something like: Select subset of above to implement/finish; do initial community check, do design basics and plan initial UXR session.
+
+## Dates
+Goal Date: <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+
+Description: <!-- Description of Deliverables-->
+
+Testing Days required: <!-- Days required at the end of development for testing -->
+
+## Success Metrics
```
Comment (oskarth) [inline ideas/99-confidence.md]: Can these literally be the KRs for this objective?

File: ideas/99-confidence.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+**TBD**
+
+95% of a group of 20-100 users surveyed - who don't have additional context beyond Status providing a p2p IM capability - using the app for an extended period of time, should answer 'yes' to the question: "Do you trust Status to deliver messages for you?" (and possibly variants of this).
+
+This is fundamentally a soft or qualitative goal. It is thus necessary but not necessarily sufficient, and additional harder numbers might be used as we develop the capability to measure this.
+
+Zero instabug reports within 30 days of alpha release
+
```
Comment (oskarth) [inline ideas/99-confidence.md]: Need some form of exit criteria. Not sure what's best here. Thoughts @rasom and @chadyj?

File: ideas/99-confidence.md (hunk 2 above)
Comment (rasom) [reply ideas/99-confidence.md]: probably 1 QA? but otherwise it's enough

**Comment by oskarth** [comment]
@rasom Please open another PR once we fleshed this out a bit, above review still stands. I know there's some loose context in misc Slack chats / google docs as well.

**Pull request merged**